hack/generators/conversion-webhook/main.go

```diff
@@ -84,6 +85,65 @@ func main() {
 	fmt.Println("Successfully finished")
 }
 
+func wrapInIfEnabled(v string) string {
+	return fmt.Sprintf("{{- if .Values.enabled }}\n%s\n{{- end }}", v)
```

Comment:
```suggestion
	return fmt.Sprintf("{{- if .Values.enabled }}\n%s\n{{- end }}\n", v)
```

to not have this mark at the end of diff 🙂 

<img width="403" height="87" alt="Image" src="https://github.com/user-attachments/assets/5d85446b-d12b-4951-8c6d-61b4365d2f77" />